src/presentation/react/components/metadata-table/MetadataTable.tsx

```diff
@@ -405,10 +405,11 @@ const MetadataTable: React.FC<MetadataTableProps> = ({
 
     useEffect(() => {
         if (model.getCollectionName() === "organisationUnits" && !filters.parents) return;
+        const fields = model.getFields();
 
         setLoading(true);
         compositionRoot.metadata
-            .list({ ...filters, filterRows, fields: model.getFields() }, remoteInstance)
+            .list({ ...filters, filterRows, fields, includeParents: true }, remoteInstance)
```

Comment:
Shouldn't it be includeParents true if model is organisationUnits and false otherwise?

Reply:
I thought the idea was to keep it as generic as possible, in case in the future any model has parents. Anyway, it should be ignored for any other model since `parents` will be undefined, right?

Reply:
If we follow that logic the `includeParents` is redundant. It could be removed completely and we would still fallback to the isNotEmpty check.

In any case, @adrianq this is not very important and the PR can be merged anyway.

Reply:
I see your point, I was thinking of another potentiall call to the use case without that option, but probably use cases do not get to reused like this.

Reply:
As discussed, as we don't have a strong opinion I would leave it for the next one changing this code.